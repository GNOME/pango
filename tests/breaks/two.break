# Example from https://gitlab.gnome.org/GNOME/pango/issues/218
# This shows difference between word start/end and boundary
gorilâ€§les
