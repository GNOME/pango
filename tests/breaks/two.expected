Text:         g o r i l Â· l e s  [0x0a] 
Breaks:     c  c c c c c c c c c       c
Whitespace:                    w       w
Words:      bs         e s     be      b
Sentences:  bs                 e       b
